scout/server/extensions/loqus_extension.py

```diff
@@ -42,92 +42,209 @@ def execute_command(cmd):
 
 class LoqusDB:
     """Interface to loqusdb from Flask
-
-    NOTE: * initialied in __init__.py called from server/extensins/__init.py__ but also
-            init_app is called from server/app.init.
-          * in practice this is a singleton class
-          * configured in `scout/server/config.py`"""
-
-    def __init__(self, loqusdb_binary=None, loqusdb_config=None, loqusdb_args=None, version=None):
```

Comment:
Removing this, as it is not used at all by the flask app, but only in tests. I've modified these tests to test the Loqus commands while being in the app context